HopsanGenerator/src/generators/HopsanGeneratorBase.cpp

```diff
@@ -1056,7 +1056,7 @@ bool HopsanGeneratorBase::copyDefaultComponentCodeToDir(const QString &path) con
     return true;
 }
 
-bool HopsanGeneratorBase::copyExternalComponentCodeToDir(const QString &destinationPath, const QStringList &externalLibraries) const
+bool HopsanGeneratorBase::copyExternalComponentCodeToDir(const QString &destinationPath, const QStringList &externalLibraries, QStringList *extraSourceFiles) const
```

Comment:
Any particular reason for using a pointer to QStringList ?

Reply:
Well the only reason was that FMI and Simulink export did not yet support extra source files. I can add support for them as well.

Reply:
I meant, why use a pointer instead of a reference like it is in most other places

Reply:
Because a reference argument can't have a default value, i.e. nullptr. But never mind, I added support for this in he other generators as well now.